tools/ci_build/github/linux/build_linux_python_package.sh

```diff
@@ -9,13 +9,26 @@ EXTRA_ARG=""
 ENABLE_CACHE=false
 # Put 3.10 at the last because Ubuntu 22.04 use python 3.10 and we will upload the intermediate build files of this
 # config to Azure DevOps Artifacts and download them to a Ubuntu 22.04 machine to run the tests.
-PYTHON_EXES=("/opt/python/cp311-cp311/bin/python3.11" "/opt/python/cp312-cp312/bin/python3.12" "/opt/python/cp313-cp313/bin/python3.13" "/opt/python/cp313-cp313t/bin/python3.13t" "/opt/python/cp310-cp310/bin/python3.10")
-while getopts "d:p:x:c:e" parameter_Option
+PYTHON_EXES=(
+  "/opt/python/cp311-cp311/bin/python3.11"
+  "/opt/python/cp312-cp312/bin/python3.12"
+  "/opt/python/cp313-cp313/bin/python3.13"
+  "/opt/python/cp313-cp313t/bin/python3.13t"
+  "/opt/python/cp310-cp310/bin/python3.10"
+  )
+while getopts "d:p::x:c:e" parameter_Option
 do case "${parameter_Option}"
 in
 #GPU|CPU|NPU.
 d) BUILD_DEVICE=${OPTARG};;
-p) PYTHON_EXES=("${OPTARG}");;
+p)
+  # If OPTARG is empty, -p was provided without an argument
+  if [ -z "${OPTARG}" ]; then
```

Comment:
In that case, why do we want to pass "-p" at all?

Reply:
We don't, but in case someone forget to put `-p` without argument. Like previously `build_linux_python_package.sh -p -d GPU` was called . In that case the value of -p was evaluated as `-d` instead of the default value.

Reply:
They are all our code. We can fix it, if we forgot doing something.  I would prefer to keep the code simpler.